Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -618,24 +618,24 @@ public bool RegisterService<T>(Type interfaceType, SupportedPlatforms supportedP
                 return false;
             }
 
-            if (interfaceType == null)
+#if !UNITY_EDITOR
```

Comment:
These changes should be in a different pull request

Reply:
That's true. How do I move them over?

Reply:
I'd suggest making a new branch from the latest development and copying over the changes.

Then reverting this file to what it was before.

Reply:
will do tomorrow

Reply:
Thanks :)